docker.go

```diff
@@ -99,5 +106,27 @@ func loadDockerImage(cli *client.Client, pathToImage string) (string, error) {
 
 	b, err := ioutil.ReadAll(ret.Body)
 
-	return string(b[:]), nil
+	// {"stream":"Loaded image: sles12/mariadb:10.0\n"}
+	var loadResponseBody LoadResponseBody
+	if err := json.Unmarshal(b[:], &loadResponseBody); err != nil {
+		return "", err
+	}
+
+	return strings.TrimSpace(loadResponseBody.Stream)[14:], nil
```

Comment:
Why is `14` hardcoded in this slice?

Reply:
What about "return strings.TrimSpace(strings.TrimPrefix(loadResponseBody.Stream, "Loaded image:")), nil

Reply:
@jordimassaguerpla: This is much better